vocode/streaming/output_device/exotel_output_device.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
```

Comment:
could you link me to some docs here re the websocket spec for exotel? i'm surprised it's so similar to Twilio

Reply:
@ajar98 Here is an article regarding exotel websocket
https://support.exotel.com/support/solutions/articles/3000108630-working-with-the-stream-and-voicebot-applet